### PR TITLE
Highlighting selected Sidebar tabs

### PR DESCRIPTION
**Pull request by Tushar3099**
Signed-off-by: Tushar3099 <tusharneogi30@gmail.com>

Fixes Issue: #400 

**Type Of Change**
----

- [x] Bug

**Changes**
----

- Highlighted selected tabs in the sidebar for a better user experience.

**Preview**
----

https://user-images.githubusercontent.com/51849085/112000483-0b62e680-8b44-11eb-8a7b-3ab7544ac1f7.mp4

### CODE REVIEW

**Comment by Tushar3099** [comment]
@muskankhedia please review my PR.

File: dashboard/v1.1/src/layouts/Sidebar.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const useStyles = makeStyles(theme => ({
   // Nested lists
   nested: {
     paddingLeft: theme.spacing(4),
-    backgroundColor: '#DCDCDC'
+    backgroundColor: 'white'
```
Comment (muskankhedia) [inline dashboard/v1.1/src/layouts/Sidebar.tsx:35]:
```suggestion
    // backgroundColor: '#DCDCDC'
```

The white colour leads to an inconsistency in the dark mode.
Screenshot 2021-03-22 at 8 46 48 AM

**Review by muskankhedia — CHANGES_REQUESTED** [review]
Can you please add some margin between the sidebar elements so the background colour can be seen effectively, especially in the case of `Test` option.

Screenshot 2021-03-22 at 8 49 54 AM

File: dashboard/v1.1/src/layouts/Sidebar.tsx (hunk 1 above)
Comment (Tushar3099) [reply dashboard/v1.1/src/layouts/Sidebar.tsx:35]: Thanks for the suggested improvements. I am doing the prescribed changes and will make a PR ASAP.

Comment (muskankhedia) [reply dashboard/v1.1/src/layouts/Sidebar.tsx:35]: @Tushar3099 I think it will be better if you can update this PR itself.

Comment (Tushar3099) [reply dashboard/v1.1/src/layouts/Sidebar.tsx:35]: Yeah sorry, I meant the same :+1: . I will update this PR itself.

**Comment by Tushar3099** [comment]
@muskankhedia
-  I have made the changes you prescribed. 
- Also, I have added a hover effect for better user interaction.

**Review by muskankhedia — APPROVED** [review]
LGTM! Thank you for the PR.

**Pull request merged**